crates/shared/src/sources/uniswap_v3/pool_fetching.rs

```diff
@@ -570,7 +571,7 @@ mod tests {
         };
 
         let serialized = serde_json::to_value(pool).unwrap();
-        assert_eq!(json, serialized);
+        assert_json_matches!(json, serialized, []);
```

Comment:
Were these the only instances where we tested a JSON for equality so for?
What was your process for finding these?

Reply:
I was searching by `json!` macro usage. Added it also into the `models` crate.